examples/destructure-tuples/destructure_tuples.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/io;
+
+type Time [int, decimal];
+
+// This function accepts a variable of type Time and converts it to seconds in decimal.
+function toSeconds(Time time) returns decimal {
+    // The tuple value is destructured into two variables: day and seconds.
+    var [day, seconds] = time;
+    decimal s = 86400d * <decimal>day;
+    s += seconds;
+    return s;
+}
+
+public function main() {
+    Time time = [1, 12];
+    Time[] timeArray = [[1, 9.2], [2, 7.1]];
+
+    // Each tuple value in `timeArray` is destructured into two variables: day and seconds.
+    foreach var [day, seconds] in timeArray {
+        io:println(`day: ${day} seconds: ${seconds}`);
+    }
+
+    decimal seconds = toSeconds(time);
+    io:println(seconds);
```

Comment:
LEt's move this to after L15. Generally better to have the variable declared close to where it is used.

Reply:
Please fix wherever applicable in the other BBEs introduced in this PR too.